.travis.yml

```diff
@@ -134,4 +134,5 @@ script:
   - bin/travis_clang.sh
   - bin/test_travis.sh
 notifications:
+  slack: ecs193:jUcubCRxGys1NdYESk9tjBWA
```

Comment:
I'd like to remove that commit now. But when I do rebase or cherry-pick I'm stuck in a series of some bad merge conflicts.